### PR TITLE
Update code-of-conduct.md

### PR DESCRIPTION
**Pull request by spiffxp**
Refer to kubernetes/community as authoritative source for code of conduct

ref: kubernetes/community#1527

### CODE REVIEW

**Review by cblecker — APPROVED** [review]
/lgtm

File: code-of-conduct.md
Comment (bacongobbler) [inline code-of-conduct.md]: This site is still live. Is this intentionally being phased out?

**Review by bacongobbler — CHANGES_REQUESTED** [review]
It looks like the Kubernetes code of conduct redirects to the CNCF code of conduct which we are already pointing to. I don't see the need for an extra level of indirection here.

File: code-of-conduct.md
Comment (spiffxp) [reply code-of-conduct.md]: I wasn't aware that it was still live, who can I talk to about whether these analytic links are actively being used and how/whether they should be put in place for each individual repo's .md files?

**Comment by spiffxp** [comment]
@bacongobbler I'm doing this under the impression that "helm" is part of "the kubernetes project" and per the referenced community issue, we'd like all repos in the kubernetes project to refer to the same code of conduct.  The extra level of indirection is there in the event that we wish to edit or alter our code of conduct without altering the CNCF's code of conduct.

File: code-of-conduct.md
Comment (bacongobbler) [reply code-of-conduct.md]: `git blame` points the finger at @philips :)

https://github.com/kubernetes/helm/commit/c97f4d7cd36d07ff36ffd30d0df0739627a95e3e#diff-d3030a18b089fdb1fbfabf6e75e4aef0

@philips do you know if this is still in use?

Comment (bacongobbler) [reply code-of-conduct.md]: following the breadcrumbs brought me back to the original commit when the kubernetes community code of conduct was created, which still brings us back to Brendan. That's as far as my google-fu got me :) https://github.com/kubernetes/kubernetes-template-project/commit/25ead97bebdff65bfc177908ac79d4f00dd90bb6

**Comment by bacongobbler** [comment]
Sounds fair. It's just awkward to be pointing at a "redirection" link from a usability standpoint. I'll hold off LGTM'ing this until we get an idea whether we should keep the analytics link, otherwise this looks good.

**Comment by mattfarina** [comment]
@bacongobbler if you look at the linked issue in the original PR you'll see that everyone is making the switch in k8s. Shouldn't Helm follow suit?

**Comment by bacongobbler** [comment]
Yes, I can see that. I wanted to give @philips an opportunity to comment on whether this analytics link is still useful for the steering committee before we went ahead and removed it.

Let's just go ahead and merge this. I'll leave it to the steering committee to figure out if it's still useful.

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**
